### PR TITLE
[onert] Simplify OperationLowerInfo usage

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit removes OperationLowerInfo and use Backend directly.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Related issue: #12130

### CODE REVIEW

**Review by ragmani — APPROVED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

File: runtime/onert/core/include/compiler/CodeMap.h [hunk 1]
```diff
@@ -17,11 +17,12 @@
...
 
-#include <unordered_map>
+#include "backend/Backend.h"
+#include "exec/FunctionSequence.h"
 #include "ir/Index.h"
 #include "ir/IOperation.h"
-#include "exec/FunctionSequence.h"
-#include "OperationLowerInfo.h"
+
+#include <unordered_map>
```
Comment (glistening) [inline runtime/onert/core/include/compiler/CodeMap.h:25]: Unnecessary header inclusion

```suggestion
```

File: runtime/onert/core/include/compiler/train/TrainableCodeMap.h [hunk 2]
```diff
@@ -17,11 +17,12 @@
...
 #define __ONERT_COMPILER_TRAIN_TRAINABLE_CODE_MAP_H__
 
-#include <unordered_map>
-#include "compiler/OperationLowerInfo.h"
+#include "backend/Backend.h"
 #include "exec/train/TrainableFnSequence.h"
 #include "ir/train/ITrainableOperation.h"
 
+#include <unordered_map>
+
```
Comment (glistening) [inline runtime/onert/core/include/compiler/train/TrainableCodeMap.h:25]:
```suggestion
```

**Review by glistening — COMMENTED** [review]
There are unnecessary inclusion of `unordered_map`.

**Comment by hseok-oh** [comment]
I'll update inclusion later.

**Pull request merged**